doc/manual/judging.rst

```diff
@@ -9,10 +9,13 @@ The flow of an incoming submission is as follows.
    checks, or accepted and stored as a *submission*.
 #. The first available *judgehost* compiles, runs and checks
    the submission. The outcome and outputs are stored as a
-   *judging* of this submission. Note that judgehosts may be
-   restricted to certain contests, languages and problems, so that it can be
-   the case that a judgehost is available, but not judging an available
-   submission.
+   *judging* of this submission. Note that it is possible to
```

Comment:
nit: I would phrase it like this:
"If parallel judging is enabled, multiple judgehosts may pick up and work on the same submission (if there is no queue of pending submissions)."